Detectors/EMCAL/base/include/EMCALBase/Geometry.h

```diff
@@ -22,9 +23,12 @@
 #include <TNamed.h>
 #include <TParticle.h>
 #include <TVector3.h>
+#include "Math/GenVector/DisplacementVector3D.h"
+#include "Math/GenVector/PositionVector3D.h"
```

Comment:
I will try to remove the headers. I guess they were leftovers from the merge. Thanks for spotting!